docs/concepts/account-linking/index.mdx

```diff
@@ -3,80 +3,120 @@ title: Account Linking (FLIP 72)
 
-</Callout>
+![Multiple parent-child accounts on Flow](resources/multiple-accounts.png)
+
+This shared control over the digital items in the in-app account enables users to establish real ownership of the items beyond the context of the app, where they can use their parent account to view inventory, take the items to other apps in the ecosystem, such as a marketplace or a game.
+
+Most importantly, users are able to do this without the need to transfer the digital items between accounts, making it seamless to continue using the original app while also using other apps.
+
+## Security Consideration with Account Linking
+
+Account linking is a very powerful feature of Cadence and thus it must be treated with care. In this document, we’ve always talked about account linking between two accounts we own, even if the child account is managed by a third-party application, that we access through an authentication mechanism, it’s still treated as our account.
+
+Creating an `AuthAccount` capability and publishing it to an account we don’t own means we are giving that account full access to our account. This should be seen as an anti-pattern.
+
+<aside>
+❗ Creating an `AuthAccount` capability and sharing it with another person account is in concept the same as giving that person your accounts private keys.
+
+</aside>
+
+Because pure account linking can be dangerous Flow introduces a Hybrid Custody contract that helps manage child accounts and access.
+
+## Hybrid Custody and Account Linking
 
-This shared control over the digital items in the in-app account enables users to establish real ownership of the items
-beyond the context of the app, where they can use their self-custody wallet to view inventory, take the items to other
-apps in the ecosystem, such as a marketplace or a game.
+Apps need assurances that their own resources are safe from malicious actors, so giving out full access might not be the form they want. Using hybrid custody contracts, the app still maintains control of their managed accounts, but they can:
 
-Most importantly, users are able to do this without the need to transfer the digital items between accounts,
-making it seamless to continue using the original app while also using other apps.
+1. Share capabilities freely, with a few built-in controls over the types of capabilities that can be returned with some helper contracts (the `CapabilityFactory`, and `CapabilityFilter`)
+2. Share additional capabilities (public or private) with a parent account via a `CapabilityDelegator` resource
 
-With account linking, developers can build walletless onboarding experiences for new users that provide a clear path to
-self-custody. Continue on to read our guides for more details on how you can use Account Linking to
-build spectacular experiences on Flow.
+Learn more about it in the Hybrid Custody documentation.
```

Comment:
Link to the docs?

Reply:
Plan to make a final PR with all the links since not all the docs were there at the time of writing.